src/firejail/dhcp.c

```diff
@@ -139,18 +140,32 @@ void dhcp_start(void) {
 	if (!any_dhcp())
 		return;
 
+	char *dhclient_path = "/sbin/dhclient";
+	struct stat s;
+	if (stat(dhclient_path, &s) == -1) {
+		dhclient_path = "/usr/sbin/dhclient";
+		if (stat(dhclient_path, &s) == -1) {
+			fprintf(stderr, "Error: dhclient was not found.\n");
+			exit(-1);
```

Comment:
Can you return `1`? Just to keep it consistent.

Reply:
Done.